### PR TITLE
CIV-5255 - CUI Welsh UI Translations 21

### PR DESCRIPTION
**Pull request by pascalkankonde**
**Before creating a pull request make sure that:**

### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-5255

### Change description ###

The scope of this ticket is to develop Welsh content translation of the following screens: 

DQ-Do you want an extra 4 weeks to settle the claim
DQ-Enter the expert's details

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/views/features/directionsQuestionnaire/request-extra-4weeks.njk [hunk 1]
```diff
@@ -33,7 +33,7 @@
           </div>
           <form method="post" novalidate autocomplete="off">
             {{ csrfProtection(csrf) }}
-            {{ yesNoRadioButton(t, 'option', 'option', '', form, '') }}
+            {{ yesNoRadioButton(t,id,name,title,form,'',{yes: 'COMMON.VARIATION_2.YES', no: 'COMMON.VARIATION_2.NO'}) }}
```
Comment (laarmada) [inline src/main/views/features/directionsQuestionnaire/request-extra-4weeks.njk:36]: are you sure the tests are passing? if you remove option from there, it doesn`t work

**Review by laarmada — CHANGES_REQUESTED** [review]

File: src/main/modules/i18n/locales/cy.json [hunk 2]
```diff
@@ -980,17 +980,17 @@
...
+      "ADD_ANOTHER_EXPERT": "Ychwanegu arbenigwr arall",
+      "ADD_EXPERT": "Ychwanegu arbenigwr arall",
+      "COST_OPTIONAL": "Amcangyfrif o’r gost (dewisol)",
+      "EMAIL_ADDRESS_OPTIONAL": "Cyfeiriad e-bost (dewisol)",
+      "FIELD_OF_EXPERTISE": "Maes arbenigedd",
+      "FIRST_NAME_OPTIONAL": "Enw cyntaf (dewisol)",
+      "LAST_NAME_OPTIONAL": "Enw olaf (dewisol)",
+      "PAGE_TITLE": "Nodwch fanylion yr arbenigwr",
+      "PHONE_OPTIONAL": "Rhif ffôn (dewisol)",
       "REMOVE_EXPERT": "Remove expert",
```
[the remark this answers is not in the document]
Comment (pascalkankonde) [reply src/main/modules/i18n/locales/cy.json:992]: Remove expert, was not provided

File: src/main/views/features/directionsQuestionnaire/request-extra-4weeks.njk (hunk 1 above)
Comment (pascalkankonde) [reply src/main/views/features/directionsQuestionnaire/request-extra-4weeks.njk:36]: I have tested, it is fine

Comment (commitBlob) [reply src/main/views/features/directionsQuestionnaire/request-extra-4weeks.njk:36]: This won't work. select an option and press save and continue, you can't get out of the page

**Review by commitBlob — CHANGES_REQUESTED** [review]

**Comment by commitBlob** [comment]
Removing myself as a reviewer as otherwise will be blocking this issue.
Pascal to refactor yesNoRadioButton to  `yesNoRadioButton(t, 'option', 'option', title ,form, '', {yes: 'COMMON.VARIATION_2.YES', no: 'COMMON.VARIATION_2.NO'})` as Sergio pointed out page won't function properly.
Remove expert translation to be requested from Josh as well.

**Review by commitBlob — COMMENTED** [review]
Removing myself as a reviewer as otherwise will be blocking this issue.
Pascal to refactor yesNoRadioButton to  `yesNoRadioButton(t, 'option', 'option', title ,form, '', {yes: 'COMMON.VARIATION_2.YES', no: 'COMMON.VARIATION_2.NO'})` as Sergio pointed out page won't function properly.
Remove expert translation to be requested from Josh as well.

Need re-review when translation is provided.

**Pull request merged**